webrtc.html

```diff
@@ -3073,44 +3073,54 @@ <h4>
-                          not implemented by the browser [= exception/throw =]
-                          a {{NotSupportedError}}. If <var>scheme name</var> is
-                          <code class="scheme">turn</code> or <code class=
-                          "scheme">turns</code>, and parsing the <var>url</var>
-                          using the syntax defined in [[!RFC7065]] fails, [=
-                          exception/throw =] a {{SyntaxError}}. If <var>scheme
-                          name</var> is <code class="scheme">stun</code> or
-                          <code class="scheme">stuns</code>, and parsing the
-                          <var>url</var> using the syntax defined in
-                          [[!RFC7064]] fails, [= exception/throw =] a
-                          {{SyntaxError}}.
-                        </p>
+                        <p>Let <var>parsedURL</var> be the result of
+                        <a data-cite="!url#concept-url-parser">parsing</a>
+                        <var>url</var>.</p>
                       </li>
                       <li>
-                        <p>
-                          If <var>scheme name</var> is <code class=
-                          "scheme">turn</code> or <code class=
-                          "scheme">turns</code>, and either of
-                          <var>server</var>.{{RTCIceServer/username}} or
-                          <var>server</var>.{{RTCIceServer/credential}} are
-                          omitted, then [= exception/throw =] an
-                          {{InvalidAccessError}}.
-                        </p>
+                        <p>If any of the following conditions apply, then [=exception/throw=] a
+                        "{{SyntaxError}}" {{DOMException}}:</p>
+                        <ul>
+                          <li><var>parsedURL</var> is failure</li>
+                          <li><var>parsedURL</var>'s [=url/scheme=] is neither `"stun"`,
+                          `"stuns"`, `"turn"`, nor `"turns"`</li>
+                          <li><var>parsedURL</var> does not have an [=url/opaque path=]</li>
+                          <li><var>parsedURL</var>'s' [=url/fragment=] is non-null</li>
+                          <li><var>parsedURL</var>'s' [=url/scheme=] is `"stun"` or `"stuns"`,
+                          and <var>parsedURL</var>'s' [=url/query=] is non-null</li>
+                        </ul>
+                      </li>
+                      <li>
+                        <p>If <var>parsedURL</var>'s [=url/scheme=] is not implemented by the
+                        user agent, then [=exception/throw=] a {{NotSupportedError}}.</p>
+                      </li>
+                      <li>
+                        <p>Let <var>hostAndPortURL</var> be result of
+                        <a data-cite="!url#concept-url-parser">parsing</a> the concatenation of
+                        `"https://"` and <var>parsedURL</var>'s [=url/path=].</p>
```

Comment:
This is a decidedldy odd way of parsing the arguments to the URL.
It assumes that 7064 and 7065 are completely consistent with the relevant parts of the URL spec. I'd prefer to have the URL spec learn to parse TURN URLs - @annevk may have opinions on the proper way to do this.